Core/src/org/sleuthkit/autopsy/commonfilesearch/CommonFilesPanel.java

```diff
@@ -202,7 +208,11 @@ private Long determineDataSourceId() {
 
             @Override
             @SuppressWarnings({"BoxedValueEquality", "NumberEquality"})
-            protected CommonFilesMetadata doInBackground() throws TskCoreException, NoCurrentCaseException, SQLException {
+            protected CommonFilesMetadata doInBackground() throws TskCoreException, NoCurrentCaseException, SQLException, EamDbException {
+                progress = ProgressHandle.createHandle(Bundle.CommonFilesPanel_search_done_searchProgress1());
+                progress.start();
+                progress.switchToIndeterminate();
+                
```

Comment:
This is interesting, I don't think we have a precedent for providing a progress bar in the lower right hand corner when doing search operations. I'm not sure the user will notice. We do have a new search feature where we provide a progress bar on the search dialog, but I don't want to undertake that additional complexity at this time. We need to think over whether this is the right approach.

Reply:
Removed for now.